setup.py

```diff
@@ -6,7 +6,7 @@
 
 
 install_requires = [
-    "Twisted>=18.9.0",
+    "Twisted>=20.3.0",
```

Comment:
What is the reason for this bump?

Reply:
Windows build is failing for 18.9.0. Trying to determine whether this is a version issue.

Reply:
I checked this and yeah, iocpsupport from Twisted < 20.3.0 fails to build on Python 3.8 so this makes sense.